### ISSUE TITLE
Dúvidas sobre os dados extraídos do Registro Civil

### ISSUE THREAD

**Issue by gjdutra** [capyvara/brazil-civil-registry-data#9]
Oi meu caro, tudo bem? Eu vi os dados que você está extraindo da base de registro civil sobre Covid-19 e gostaria de tirar algumas dúvidas com você. Você saberia me informar porque tem a divisão das mortes de Covid-19 em três variáveis:
- deaths_stroke_covid19 - Number of stroke deaths with Covid-19; 
- deaths_heart_attack_covid19 - Number of heart attack deaths with Covid-19;  e
- deaths_covid19.
Para encontrar o número de mortes por Covid-19 de cada região é preciso somar as três variáveis acima? É possível conseguir a mortalidade por Covid-19 associada a outros fatores como diabetes? Abraços, parabéns pelo trabalho e muito obrigado pela atenção.

**Comment by capyvara** [comment]
Sim, é necessário somar, olhem as notas no fim da página deles de como é separado isso:
https://transparencia.registrocivil.org.br/especial-covid

Mapeamento: https://transparencia.registrocivil.org.br/covid/dicionario_cardiaco.pdf

Eu costumo agrupar assim:
```
df2["others"] = grp["deaths_others"] + grp["deaths_indeterminate"]
df2["stroke"] = grp["deaths_stroke"]
df2["septicemia"] = grp["deaths_septicemia"]
df2["respiratory_failure"] = grp["deaths_respiratory_failure"]
df2["pneumonia"] = grp["deaths_pneumonia"]
df2["heart"] = grp["deaths_cardiopathy"] + grp["deaths_cardiogenic_shock"] + grp["deaths_heart_attack"] + grp["deaths_sudden_cardiac"]
df2["sars"] = grp["deaths_sars"]
df2["covid19"] = grp["deaths_covid19"] + grp["deaths_stroke_covid19"] + grp["deaths_heart_attack_covid19"]
```

**Comment by gjdutra** [comment]
Muito obrigado pelo retorno :). 

Eu fiz a soma dos óbitos de Covid-19 e fiquei com mais uma dúvida. Consegui encontrar um total de 196154 mortes de Covid-19 no banco de dados, referente ao ano de 2020. Porém, o gráfico do portal de transparência do Registro Civil demonstra um total de 197370 (https://transparencia.registrocivil.org.br/especial-covid). Você sabe porque pode estar acontecendo essa diferença? É possível que essa diferença esteja associada a missings de preenchimento de variáveis como sexo ou raça/cor nas declarações de óbito?

**Comment by capyvara** [comment]
Eu somei o meu aqui agora, versão com os estados de [hoje de manhã](https://github.com/capyvara/brazil-civil-registry-data/blob/7f6bc5874fdbb1f625914077a7ecda29927ae07c/civil_registry_covid_states.csv) e deu 197372, covid sendo a soma mencionada acima.

O de cidades não contém todas as cidades do Brasil, seria infactível fazer o scrap pelo tempo e erros que ocorrem no processo.
